Source/CombatExtended/CombatExtended/Defs/AmmoDef.cs

```diff
@@ -22,6 +22,8 @@ public class AmmoDef : ThingDef
         // mortar ammo should still availabe when the ammo system is off
         public bool isMortarAmmo = false;
 
+        public List<string> ammoTags;
```

Comment:
unused?

Reply:
no, so the team can start working on the ammo tag system, if it's not populated i can do anything with it thus this is the first step.